### PR TITLE
Add Trolleybus Support

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
We forgot to add trolleybus support. This PR adds trolleybus support across the entire project. Adds unit tests to make sure it worked.

### CODE REVIEW

**Review by AdrianaCeric — APPROVED** [review]
Trolleybuses are important

File: packages/core-utils/src/route.ts [hunk 1]
```diff
@@ -184,6 +184,7 @@ function getSortValues(
 const modeComparatorValue = {
   SUBWAY: 1,
   TRAM: 2,
+  TROLLEYBUS: 9,
```
Comment (amy-corson-ibigroup) [inline packages/core-utils/src/route.ts:187]: Wait shouldn't this be 11? In the GTFS spec trolleybus is 11, doesn't OTP follow the spec

File: packages/core-utils/src/route.ts [hunk 2]
```diff
@@ -207,7 +208,8 @@ const routeTypeComparatorValue = {
   // TODO: 11 and 12 are not a part of OTP as of 2019-02-14, but for now just
   // associate them with bus/rail.
   11: modeComparatorValue.BUS, // - Trolleybus.
-  12: modeComparatorValue.RAIL // - Monorail.
+  12: modeComparatorValue.RAIL, // - Monorail.
+  13: modeComparatorValue.TROLLEYBUS
```
Comment (amy-corson-ibigroup) [inline packages/core-utils/src/route.ts:212]: Lines 210 and 212 both seem to be accounting for Trolleybuses here?

**Review by amy-corson-ibigroup — COMMENTED** [review]
Quick question!!

File: packages/core-utils/src/route.ts (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/core-utils/src/route.ts:187]: I don't think this has anything to do with OTP mode numbers. This has to do with determining route sort order

File: packages/core-utils/src/route.ts (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/core-utils/src/route.ts:212]: The comment is wrong, but only 212 accounts for trolleybus

Comment (amy-corson-ibigroup) [reply packages/core-utils/src/route.ts:212]: Could we remove the wrong comment

**Review by amy-corson-ibigroup — APPROVED** [review]

File: packages/core-utils/src/route.ts (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/core-utils/src/route.ts:212]: I'm worried about what information we might lose... These comments are very old!

**Pull request merged**